index.md

```diff
@@ -46,25 +46,18 @@ For more options:
 
     $ stack exec -- gibbon -h
 
-
-<div id="publications">
-
-<h2>Publications</h2>
-
-<table>
-<tr>
-<td>ECOOP'17</td>
-<td><b> Compiling Tree Transforms to Operate on Packed Representations:<br/></b> Michael Vollmer, Sarah
-Spall, Buddhika Chamith, Laith Sakka, Chaitanya Koparkar, Milind Kulkarni, Sam Tobin-Hochstadt, Ryan Newton [<a href="http://drops.dagstuhl.de/opus/volltexte/2017/7273/pdf/LIPIcs-ECOOP-2017-26.pdf" target="_blank">PDF</a>]</td>
-</tr>
-<tr>
-<td>PLDI'19</td>
-<td><b> LoCal: A Language for Programs Operating on Serialized Data:<br/></b> Michael Vollmer, Chaitanya Koparkar, Mike Rainey, Laith Sakka, Milind Kulkarni, Ryan R. Newton [<a href="http://recurial.com/pldi19main.pdf" target="_blank">PDF</a>]</td>
-</tr>
-<tr>
-<td>ICFP'21</td>
-<td><b> Efficient Tree-Traversals: Reconciling Parallelism and Dense Data Representations:<br/></b> Chaitanya Koparkar, Mike Rainey, Michael Vollmer, Milind Kulkarni, Ryan R. Newton [<a href="./public/icfp21.pdf" target="_blank">PDF</a>]</td>
-</tr>
-</table>
-
-</div>
+## Publications
+
+|||
+|--- |--- |
+|ECOOP'17|**Compiling Tree Transforms to Operate on Packed Representations:** <br/> Michael Vollmer, Sarah Spall, Buddhika Chamith, Laith Sakka, Chaitanya Koparkar, Milind Kulkarni, Sam Tobin-Hochstadt, Ryan Newton [[PDF][ecoop17]]|
+|PLDI'19|**LoCal: A Language for Programs Operating on Serialized Data:** <br/> Michael Vollmer, Chaitanya Koparkar, Mike Rainey, Laith Sakka, Milind Kulkarni, Ryan R. Newton [[PDF][pldi19]]|
+|ICFP'21|**Efficient Tree-Traversals: Reconciling Parallelism and Dense Data Representations:** <br/> Chaitanya Koparkar, Mike Rainey, Michael Vollmer, Milind Kulkarni, Ryan R. Newton [[PDF][icfp21]]|
+|ISMM'24|**Garbage Collection for Mostly Serialized Heaps:** <br/> Chaitanya Koparkar, Vidush Singhal, Aditya Gupta, Mike Rainey, Michael Vollmer, Artem Pelenitsyn, Sam Tobin-Hochstadt, Milind Kulkarni, Ryan R. Newton [[PDF][ismm24]]|
+|ECOOP'24|**Optimizing Layout of Recursive Datatypes with Marmoset:** <br/> Vidush Singhal, Chaitanya Koparkar, Joseph Zullo, Artem Pelenitsyn, Michael Vollmer, Mike Rainey, Milind Kulkarni, Ryan R. Newton [[PDF][ecoop24]]|
```

Comment:
Fix author order for ECOOP 24

Reply:
What's wrong with the author order?

Reply:
check preprint: https://arxiv.org/pdf/2405.17590. MK is last not RN.

Reply:
My bad! Fixed, thank you.